examples/wagmi/tests/e2e/specs/tests.js

```diff
@@ -0,0 +1,36 @@
+describe('connect wallet spec', () => {
+    before(() => {
+      console.log('Visiting /');
+      cy.visit('/');
+      console.log('Done visiting');
+    });
+
+    /*
+    it('should connect wallet with success', () => {
+      console.log('Clicking button');
+      cy.get('#connect-injected-sapphire').click();
+      console.log('Accept Metamask');
+      cy.acceptMetamaskAccess();
+      console.log('Checking...');
+      cy.get('#accountAddress').should(
+        'have.text',
+        '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+      );
+    });
+    */
```

Comment:
Dead code. You can remove it with commit, so it stays in history. But I wouldn't commit this to the repo.